app/src/androidTest/java/br/com/simplepass/loadingbutton/CircularProgressButtonTest.kt

```diff
@@ -0,0 +1,108 @@
+package br.com.simplepass.loadingbutton
+
+import android.app.Application
+import android.content.Intent
+import android.view.View
+import android.view.View.VISIBLE
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import androidx.test.espresso.matcher.ViewMatchers.isEnabled
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.espresso.matcher.ViewMatchers.withEffectiveVisibility
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.rule.ActivityTestRule
+import androidx.test.runner.AndroidJUnit4
```

Comment:
This version is deprecated. 

Use `androidTestImplementation 'androidx.test.ext:junit:1.1.0'` instead of `androidTestImplementation 'androidx.test:runner:1.1.0'`. 

Take a look here: https://stackoverflow.com/questions/52776716/androidjunit4-class-is-deprecated-how-to-use-androidx-test-ext-junit-runners-an

Reply:
+1. FWIW in my experience, the `RunWith` annotation is unnecessary for a test like this.